gmaps/job.go

```diff
@@ -33,11 +33,19 @@ func NewGmapJob(id, langCode, query string, maxDepth int, extractEmail bool) *Gm
 		id = uuid.New().String()
 	}
 
+	mapURL := fmt.Sprintf("https://www.google.com/maps/search/%s", query)
+	if geoCoordinates != "" {
+		mapURL = fmt.Sprintf("https://www.google.com/maps/search/%s/@%s", query, strings.ReplaceAll(geoCoordinates, " ", ""))
```

Comment:
this url seems not to do the expected.

but this looks to work:

https://www.google.com/maps/search/restaurant/?ll=37.97945,23.71622&z=20

can you please verify and adjust?

Reply:
The search feature does not work with your URL

Reply:
I checked again, and my URL only works with the zoom parameter. But yours does not work. I will adjust the code.

Reply:
so we will have only the zoom right?